src/data/reducers/filters.js

```diff
@@ -24,11 +24,11 @@ const reducer = (state = initialState, action) => {
       };
     case GOT_GRADES: {
       const { assignment } = state;
-      const { label, type } = assignment || {};
+      const { id, type } = assignment || {};
       if (!type) {
         const relevantAssignment = getAssignmentsFromResultsSubstate(action.grades)
           .map(chooseRelevantAssignmentData)
-          .find(assig => assig.label === label);
+          .find(assig => assig.id === id);
```

Comment:
...and here...now I'm worried that we copypasta'd this everywhere a while back and never caught it.  Maybe it'd be easier to not use this abbreviation?